exporters/prometheus/src/exporter_utils.cc

```diff
@@ -147,10 +170,10 @@ std::vector<prometheus_client::MetricFamily> PrometheusExporterUtils::TranslateT
           }
           else
           {
-            sum = nostd::get<int64_t>(histogram_point_data.sum_);
+            sum = static_cast<double>(nostd::get<int64_t>(histogram_point_data.sum_));
```

Comment:
Not related to this PR, but there is possibility of precision loss while casting from `int64_t` to `double` for larger values ( > 2^52) of `sum`. Good to have a issue to track this - to check for any such loss and log a warning.

Reply:
Thanks, I add this static cast because it will trigger a warning with clang 16 and `-Werror` will make it a error.
BTW:Do you think think it may have epsilon problem in the code `metric->histogram.sample_count = static_cast<std::uint64_t>(values[1]);` in `PrometheusExporterUtils::SetValue` ?